src/amo/components/AddonDetail.js

```diff
@@ -47,10 +47,19 @@ export class AddonDetailBase extends React.Component {
     OverallRating: DefaultOverallRating,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = { mounted: false };
+  }
+
+  componentDidMount() {
+    // Disabling react/no-did-mount-set-state because it is to prevent additional renders, but
+    // that's exactly what we want in this case (use background-image with JS).
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ mounted: true });
```

Comment:
Can you help me understand why this is needed? Why is the `<img>` tag needed at all? For server side rendering?